### ISSUE TITLE
Opening any File Makes it Dirty Immediately

### ISSUE THREAD

**Issue by tsmaeder** [eclipse-theia/theia#6932]
### Description
When I open any file, it is marked immediately dirty

### Reproduction Steps
1. start the electron example
2. Open the theia source code folder
3. Open any file (I tried .gitpod.yml, package.json and README.md
4. Observe: all editors are marked dirty immediately, even though I did not change anything

**OS and Theia version:**
Windows 10, Theia master as of Jan. 21.

**Diagnostics:** 
No logs are written to the browser console when opening the file
On the terminal, I get 
```
root INFO [nsfw-watcher: 16768] Started watching: c:\Users\thomas\code\theia\packages\process\compile.tsconfig.json
```

**Comment by akosyakov** [comment]
Could you check what is the difference? Maybe you have some extension installed which auto formats the file? editor-config like to add whitespace at the end of the file or remove, not sure what is our policy there

**Comment by tsmaeder** [comment]
No, I pulled the latest changes to master and built them unchanged. There is something odd, though: while the bug reproduces 100% when opening the theia source code, I cannot reproduce it with a smaller, contrived example.

**Comment by kittaakos** [comment]
Yeah, it's a Windows thing. As far as I remember, the same with the browser-example.

**Comment by tsmaeder** [comment]
@kittaakos so do you know what the problem is?

**Comment by kittaakos** [comment]
No, I don't 😕
